app/styles/_mixins.scss

```diff
@@ -37,3 +37,17 @@
   background-color: $background-color;
   color: $color;
 }
+
+//BREAKPOINTS
+@mixin breakpoint($point) {
```

Comment:
I also found this yesterday http://krasimirtsonev.com/blog/article/SASS-mixin-for-media-queries or this one (which is pretty close to Nick's suggestion): https://github.com/sass/sass/issues/554#issuecomment-18307019